ftw/book/keyword/config.py

```diff
@@ -11,8 +11,10 @@ def jsonConfiguration(self, *args, **kwargs):
         config = json.loads(config)
         if u'theme_advanced_buttons1' in config:
             # Plone >= 4.3
-            config[u'theme_advanced_buttons1'] += u',tinymce_keyword'
+            config[u'theme_advanced_buttons1'] += (u',tinymce_keyword,'
+                                                   'tinymce_footnote,')
         else:
             # Plone <= 4.2
             config[u'buttons'].append(u'tinymce_keyword')
+            config[u'buttons'].append(u'tinymce_footnote')
```

Comment:
Please move this config out of the `ftw.book.keyword`-Package, since it is no longer just "keyword" but now also "footnote".
I'd move it to `ftw.book.tinymce` :wink: